Ginger/GingerCoreNET/GeneralLib/EmailReadGmailOperations.cs

```diff
@@ -107,6 +112,42 @@ public Task ReadEmails(EmailReadFilters filters, EmailReadConfig config, Action<
             return Task.CompletedTask;
         }
 
+        private bool DoesSatisfyAttachmentFilter(MimeMessage message, EmailReadFilters filters)
+        {
+            IEnumerable<string> expectedContentTypes = null;
```

Comment:
Please calculate expectedContentTypes list once and reuse it instead of calculating it again

Reply:
done